maintenance/kubernetes/aks-1.22.md

```diff
@@ -0,0 +1,186 @@
+         The cluster is already on version 1.22 and is not in a failed state. No operations will occur when upgrading to the same version if the cluster is not in a failed state.
+         Since control-plane-only argument is not specified, this will upgrade the control plane AND all nodepools to version 1.22. Continue? (y/N): y
+
+      $ az aks show --resource-group prodpublick8s --name prodpublick8s --output table
+      Name             Location    ResourceGroup              KubernetesVersion    CurrentKubernetesVersion    ProvisioningState    Fqdn
+      ---------------  ----------  -------------------------  -------------------  --------------------------  -------------------  --------------------------------------------------
+      temp-privatek8s  eastus2     prodpublick8s  1.22                 1.22.6                      Succeeded            temp-privatek8s-dns-c5f4426a.hcp.eastus2.azmk8s.io
+         
+         
+         We did hit a problem with PVC and PV as the parameter "secretNamespace" was not provided by our helm definition the secrets were not in the correct Namespace.
+         https://github.com/jenkins-infra/helm-charts/pull/158
+         
+         
+         - [x] Close maintenance windows in status (<https://github.com/jenkins-infra/status/pull/...>)
+         - [x] Write post mortem for what went wrong (if needed)
+
+        - [x] Expected downtimes: none
+
+### Issues Met during This Upgrade
+        
+
+
+#### Persistent Volumes of kind "Azurefile"
+
+ERROR : we had to specify the namespace where the SAS token lives, otherwises Kubernetes searches on the default namespace.
+        
+as per the AKS CSI for Kube 1.22+ (ref. jenkins-infra/helpdesk#2930), we have to specify the namespace where the SAS token lives, otherwises Kubernetes searches on the default namespace.
+
+Same hotfix as hotfix(ldap) specify secretNamespace for ldap-backup PV #158 but for the release.ci.jenkins.io environment
+Long term fix: as Upgrade to Kubernetes 1.22 helpdesk#2930 will underline, we need to check all of the PVs to use the latest CSI driver, or at least patch all the remaining azurefile PVs.
+For the record, we had to set the replica count to 0 on the pods in order to delete the PVC AND PV to apply the SecretNameSpace
+
+      ### TEMPLATE EMAIL MAILING LIST : 
+
+        Hello everybody,
+
+        We’ll do an AKS clusters upgrade from 1.XX to 1.YY this FULL DATE, starting at XX:00 UTC.
+        See more at https://github.com/jenkins-infra/helpdesk/issues/....
+        No outage is expected, but there could be an impact (1-2 min outage) on the following services if there is an unexpected upgrade issue:
+          accounts.jenkins.io
+          beta.accounts.jenkins.io
+          customize.jenkins.io
+          get.jenkins.io
+          incrementals.jenkins.io
+          infra.ci.jenkins.io
+          javadoc.jenkins.io
+          jenkins-wiki-exporter.jenkins.io…
+
+
+        The Infra Team
```

Comment:
````suggestion
### Template Email Mailing List: 

```
Hello everybody,

We’ll do an AKS clusters upgrade from 1.XX to 1.YY this FULL DATE, starting at XX:00 UTC.
See more at https://github.com/jenkins-infra/helpdesk/issues/....
No outage is expected, but there could be an impact (1-2 min outage) on the following services if there is an unexpected upgrade issue:
  accounts.jenkins.io
  beta.accounts.jenkins.io
  customize.jenkins.io
  get.jenkins.io
  incrementals.jenkins.io
  infra.ci.jenkins.io
  javadoc.jenkins.io
  jenkins-wiki-exporter.jenkins.io…

The Infra Team
```
````